reana_workflow_controller/rest/utils.py

```diff
@@ -479,16 +479,22 @@ def list_files_recursive_wildcard(directory_path, path):
        error messages.
     """
     paths, posix_dir_prefix = get_files_recursive_wildcard(directory_path, path)
-    return [
-        {
-            "name": str(path.relative_to(posix_dir_prefix)),
-            "size": path.stat().st_size,
-            "last-modified": datetime.fromtimestamp(path.stat().st_mtime).strftime(
-                WORKFLOW_TIME_FORMAT
-            ),
-        }
-        for path in paths
-    ]
+    for path in paths:
+        raw_size = path.stat().st_size
+        return [
```

Comment:
Beware, we cannot `return` from within the `for` loop, because this would show only one file and not continue with all the matched ones, for example if a user asks for `reana-client ls '*/*'`.